snapshots/storage/metastore_test.go

```diff
@@ -198,75 +200,76 @@ var baseInfo = map[string]snapshots.Info{
 }
 
 func assertNotExist(t *testing.T, err error) {
-	if err == nil {
-		t.Fatal("Expected not exist error")
-	}
-	if !errdefs.IsNotFound(err) {
-		t.Fatalf("Expected not exist error, got %+v", err)
-	}
+	t.Helper()
+	assert.Assert(t, err != nil)
+	assert.Assert(t, errdefs.IsNotFound(err), "got %+v", err)
 }
 
 func assertNotActive(t *testing.T, err error) {
-	if err == nil {
-		t.Fatal("Expected not active error")
-	}
-	if !errdefs.IsFailedPrecondition(err) {
-		t.Fatalf("Expected not active error, got %+v", err)
-	}
+	t.Helper()
+	assert.Assert(t, err != nil)
+	assert.Assert(t, errdefs.IsFailedPrecondition(err), "got %+v", err)
 }
 
 func assertNotCommitted(t *testing.T, err error) {
-	if err == nil {
-		t.Fatal("Expected active error")
-	}
-	if !errdefs.IsInvalidArgument(err) {
-		t.Fatalf("Expected active error, got %+v", err)
-	}
+	t.Helper()
+	assert.Assert(t, err != nil)
```

Comment:
Why not nil error here?

Reply:
This is the opposite of `NilError`. This is `assert.NotNil()`

Normally these would check that the error has some specific type or message, but I'm not sure what that would be, so I left it as is for now.